src/api/protectedApiClient.ts

```diff
@@ -50,9 +74,9 @@ const getRequestStatuses = (): Promise<PersonalRequest[]> => {
 };
 
 const makePFRequest = (): Promise<void> => {
-  return AppAxiosInstance.post(ProtectedApiClientRoutes.MAKE_PF_REQUEST)
-    .then(() => {})
-    .catch((err) => err);
+  return AppAxiosInstance.post(ProtectedApiClientRoutes.MAKE_PF_REQUEST).catch(
+    (err) => err,
+  );
```

Comment:
Why this change?

Reply:
![image](https://user-images.githubusercontent.com/16907452/114113273-3c654a00-98ac-11eb-9860-917b93a2fb3e.png)

We didn't add these lines, we just got rid of the .then because it was complaining. Totally down to change this to whatever it should be